### PR TITLE
 RT-1.34 admin_distance_test changes incorporated

### PR DESCRIPTION
**Pull request by ampattan**
-Changes in metadata.textproto
    -vender check added for NOKIA
    -Added deviations `explicit_interface_in_default_vrf & interface_enabled`

-Changes in admin_distance_test.go
    -Enabled "configureRoutePolicy" configuration for ebgp.
    -Added interface configuration for port3

RoutePolicy needed for implementations compliant with [RFC8212](https://datatracker.ietf.org/doc/rfc8212/)
that prescribes default EBGP import/export behaviour as REJECT

Refer PR - https://github.com/openconfig/featureprofiles/pull/1333

### CODE REVIEW

File: feature/bgp/admin_distance/otg_tests/admin_distance_test/admin_distance_test.go [hunk 1]
```diff
@@ -91,7 +92,7 @@ func TestAdminDistance(t *testing.T) {
 		ts.MustAdjacency(t)
 	})
 
-	setupEBGPAndAdvertise(t, ts)
+	setupEBGPAndAdvertise(t, ts.DUT, ts)
```
Comment (self-maurya) [inline feature/bgp/admin_distance/otg_tests/admin_distance_test/admin_distance_test.go:95]: `ts` object already has DUT instance(ts.DUT), you don't have to pass separately

Comment (ampattan) [reply feature/bgp/admin_distance/otg_tests/admin_distance_test/admin_distance_test.go]: comment addressed

**Comment by nsadhasivam** [comment]
Fails with Waiting for adjacency to form: context deadline exceeded

at https://source.corp.google.com/piper///depot/google3/third_party/openconfig/featureprofiles/feature/bgp/admin_distance/otg_tests/admin_distance_test/admin_distance_test.go;l=91;ws=nsadhasivam%2F1234?snapshot=10

**Review by nsadhasivam — APPROVED** [review]
LGTM!

passes on nokia

**Pull request merged**